### PR TITLE
Introduce static analysis

### PR DESCRIPTION
**Pull request by szepeviktor**
Hello silverstripers! :)

This is the first round of adding another 100 pair of 👀 (phpstan) to your project.
I do **like** your modern technology but there are dirt spots.
 
What do you think?

### CODE REVIEW

File: src/Control/IPUtils.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-namespace SilverStripe\Control\Util;
```
Comment (szepeviktor) [inline src/Control/IPUtils.php]: After a 🛰 spacewalk return to your place!

File: src/Dev/DebugView.php [hunk 2]
```diff
@@ -86,10 +82,6 @@ class DebugView
             'title' => 'User Deprecated',
             'class' => 'notice'
         ),
-        E_CORE_ERROR => array(
```
Comment (szepeviktor) [inline src/Dev/DebugView.php]: These 2 are doubles.

**Comment by szepeviktor** [comment]
BTW This is *some* of the Level 0 errors. Please consider going up to Level 4 (or 7)

**Comment by robbieaverill** [comment]
Hey @szepeviktor, thank you for the pull request! All the changes look good to me, the only thing I'm hesitant about is introducing phpstan. We [discussed this](https://github.com/silverstripe/silverstripe-framework/issues/7899) last year and at the time decided not to. Moving forwards, it might be worth reopening that RFC again since I think phpstan offers a lot, but for now perhaps you could remove it from composer.json and keep the rest of your changes? Thanks very much!

File: src/Control/IPUtils.php (hunk 1 above)
Comment (ScopeyNZ) [reply src/Control/IPUtils.php]: We'd have to target `master` with this change.

Comment (szepeviktor) [reply src/Control/IPUtils.php]: okay, please merge #9039

**Comment by szepeviktor** [comment]
Case closed :)

**Pull request closed without merging**